packages/carbon-graphs/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@cerner/carbon-graphs",
+  "main": "lib/Graphs.js",
+  "private": true,
+  "version": "0.0.0",
```

Comment:
Keep this version the same as the last released version of carbon-graphs

Reply:
https://github.com/cerner/terra-graphs/pull/5/commits/a331e749bc1cbe1d46395138900962a2d11a1425